indico/modules/events/papers/client/style/papers.scss

```diff
@@ -0,0 +1,10 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+.ui.page.dimmer {
+    z-index: 100 !important; // Make it less visible than Indico error-dialog
```

Comment:
why do we need this? shouldn't the error dialog (which also uses a dimmer) be on top since it's rendered later?

Reply:
Nope if you already have a SUIR dialog which has z-index of 101